wagtail/core/utils.py

```diff
@@ -220,8 +220,17 @@ def get_content_languages():
     if content_languages is None:
         # Default to a single language based on LANGUAGE_CODE
         default_language_code = get_supported_language_variant(settings.LANGUAGE_CODE)
+        try:
+            language_name = languages[default_language_code]
+        except KeyError:
+            # get_supported_language_variant on the 'null' translation backend (used for
+            # USE_I18N=False) returns settings.LANGUAGE_CODE unchanged without accounting for
+            # language variants (en-us versus en), so retry with the generic version.
+            default_language_code = default_language_code.split("-")[0]
+            language_name = languages[default_language_code]
```

Comment:
Since the display name isn't really critical, maybe we should also change this like to `languages.get(default_language_code, language_code)`.

The user will see a language code instead of the display name, but I think this would be more clear what's going on than the error?

Reply:
Good idea! It was bugging me slightly that we weren't handling LANGUAGE_CODEs not defined in LANGUAGES, but it looked like this was going to be a bit too much of a rabbit-hole to address here. This seems like a nice way around that - the display name shouldn't be a big issue, since this is mainly catering for WAGTAIL_I18N_ENABLED = False setups where we're presumably never displaying language names anyhow.

I've had to adjust things slightly so that it didn't the 'WAGTAIL_CONTENT_LANGUAGES is a subset of LANGUAGES' check immediately afterwards: 17b1f594773d6dafbfe38a6666acd1e1c14217b4